plugins/in_kubernetes_events/kubernetes_events_conf.h

```diff
@@ -38,7 +38,7 @@
 #define K8S_EVENTS_KUBE_TOKEN          "/var/run/secrets/kubernetes.io/serviceaccount/token"
 #define K8S_EVENTS_KUBE_CA             "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 
-#define K8S_EVENTS_RA_TIMESTAMP        "$metadata['creationTimestamp']"
+#define K8S_EVENTS_RA_TIMESTAMP        "lastTimestamp"
```

Comment:
isn't this just using that text instead of a record accessor pattern ?

Reply:
It seems to work with either `lastTimestamp` or `$lastTimestamp` as lastTimestamp is a top-level field. 

If it were not able to find the data i'd expect to see this error in the log: https://github.com/fluent/fluent-bit/blob/17c2ca2c382362216a0bfa74c6bd1bdd1d16ba54/plugins/in_kubernetes_events/kubernetes_events.c#L512-L516

Using a stdout output also looks correct to me:
```
[40] k8s_events: [[1703206505.000000000, {}], {"metadata"=>{"name"=>"storage-provisioner.17a29100c848d8b6", "namespace"=>"kube-system", "uid"=>"dd4006f8-2916-42fb-bc92-dc0a424978a3", "resourceVersion"=>"130490", "creationTimestamp"=>"2023-12-22T00:55:05Z", "managedFields"=>[{"manager"=>"kubelet", "operation"=>"Update", "apiVersion"=>"v1", "time"=>"2023-12-22T00:55:05Z", "fieldsType"=>"FieldsV1", "fieldsV1"=>{"f:count"=>{}, "f:firstTimestamp"=>{}, "f:involvedObject"=>{}, "f:lastTimestamp"=>{}, "f:message"=>{}, "f:reason"=>{}, "f:reportingComponent"=>{}, "f:reportingInstance"=>{}, "f:source"=>{"f:component"=>{}, "f:host"=>{}}, "f:type"=>{}}}]}, "involvedObject"=>{"kind"=>"Pod", "namespace"=>"kube-system", "name"=>"storage-provisioner", "uid"=>"a5f87337-043b-47b5-8a9c-0c603c15a01e", "apiVersion"=>"v1", "resourceVersion"=>"84639", "fieldPath"=>"spec.containers{storage-provisioner}"}, "reason"=>"BackOff", "message"=>"Back-off restarting failed container storage-provisioner in pod storage-provisioner_kube-system(a5f87337-043b-47b5-8a9c-0c603c15a01e)", "source"=>{"component"=>"kubelet", "host"=>"minikube"}, "firstTimestamp"=>"2023-12-20T14:37:21Z", "lastTimestamp"=>"2023-12-22T00:55:05Z", "count"=>15, "type"=>"Warning", "eventTime"=>nil, "reportingComponent"=>"kubelet", "reportingInstance"=>"minikube"}]
```

The documentation mentions we should use a prefixed `$` so if you'd like me to update this I can push another commit, just let me know!

Reply:
Actually, I just went ahead and updated this as it would bug me if I wasn't following the documentation :). I also updated the docs pr at the same time: https://github.com/fluent/fluent-bit-docs/pull/1274